src/locales/en-us/common/index.yaml

```diff
@@ -14,6 +14,7 @@ common:
   copySuccessText: 'Copied!'
   copyKubernetesText: 'Copy as Kubernetes'
   copyKubernetesShortText: 'as k8s'
+  copyKubernetesTooltip: 'Copy as Kubernetes'
```

Comment:
Is this needed? I would have thought that whatever text we use in the tooltip should be the text that we visually hide? Actually how accessible are our tooltips, do you know?

Reply:
What’s the point of the tooltip if it’s the exact same content as the button text?

Reply:
> What’s the point of the tooltip if it’s the exact same content as the button text?

Maybe I'd misunderstood but isn't the `copyKubernetesText` visually hidden? This is also why I was asking about the accessibility of the tooltip, if its accessible enough theres an argument to say we should try to avoid it saying "Copy as Kubernetes Copy as Kubernetes" (i.e. repeating the same text twice).

I know we aren't super hot on accessibility at the moment, but I think we are starting to get to a point where we can begin to pay more attention to these sorts of things.

I think for the moment, is it worth just reusing the `copyKubernetesText` for the text for the toolip? I suppose that is the root of my querying.

Reply:
I’ve changed this around.